packages/connect-web/package.json

```diff
@@ -47,7 +47,6 @@
         "@trezor/utils": "workspace:*"
     },
     "devDependencies": {
-        "@babel/preset-typescript": "^7.24.7",
```

Comment:
sometimes we need to duplicate dependencies in sub packages so that it works with yarn focus. not sure if this is also the case